cmake/modules/FindCdio.cmake

```diff
@@ -11,6 +11,7 @@
 
 if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_CDIO libcdio>=0.80 QUIET)
```

Comment:
```suggestion
  pkg_check_modules(PC_CDIO libcdio>=2.1.0 QUIET)
```

Reply:
Is `libcdio` below 2.1.0 really bad? 
Current and previous Debian, Fedora 32 and many other distros doesn't have 2.1.0

Reply:
I repeat: cdio++ is provided by cdio, so we must use the same version. If you manage to install 0.80 and 2.1.0 in parallel, you do something seriously wrong.

Reply:
Removed